pkg/sql/opt/exec/execbuilder/testdata/fk

```diff
@@ -959,7 +959,7 @@ render                    ·                   ·
            │    │         filter              (a_y IS NOT NULL) AND (a_x IS NOT NULL)
            │    └── scan  ·                   ·
            │              table               b@idx
-           │              spans               /!NULL-
```

Comment:
Hmm... I'm not sure, are we making things worse for no reason by redacting this information? You could have "1 span" that's basically the entire table, but you wouldn't know...